### PR TITLE
Fix the cleanup step in chipseq Docker image

### PR DESCRIPTION
**Pull request by mstajdohar**

### CODE REVIEW

File: resolwe_bio/docker_images/chipseq/Dockerfile [hunk 1]
```diff
@@ -84,9 +84,11 @@ RUN apt-get update && \
       libboost-timer-dev \
       libgoogle-perftools-dev \
       libtbb-dev && \
-
-    # Cleanup
+    apt-get autoremove --purge -y
     apt-get clean && \
-    rm -rf /var/lib/apt/lists/*
+    rm -rf /var/lib/apt/lists/* && \
+    rm -rf /var/lib/apt/lists/* && \
```
Comment (kostko) [inline resolwe_bio/docker_images/chipseq/Dockerfile]: Remove one of the lines above.

Comment (mstajdohar) [inline resolwe_bio/docker_images/chipseq/Dockerfile]: Fixed, thanks.

**Comment by kostko** [comment]
Why are there some unrelated commits now? They weren't here before.

**Comment by mstajdohar** [comment]
We need to accept six PRs by Monday, preferably today. All have been reviewed, accepted and tested before (except for this one that has not been tested yet but does not influence any tests). Running tests again separately on each would be a waste of time and resources (and not feasible in one day). As a workaround I have rebased the six PRs in a chain.

**Comment by tjanez** [comment]
> Why are there some unrelated commits now? They weren't here before.

> We need to accept six PRs by Monday, preferably today. All have been reviewed, accepted and tested before (except for this one that has not been tested yet but does not influence any tests). Running tests again separately on each would be a waste of time and resources (and not feasible in one day). As a workaround I have rebased the six PRs in a chain and canceled tests of the intermediated PRs.

Ohh, this is really ugly... @mstajdohar, we need to dedicate some resources to fix the tests taking too much time. Otherwise, we'll always have to make ugly work-around like this one...

**Comment by mstajdohar** [comment]
The tests for resolwe-bio are really huge pain.

**Pull request merged**